src/wasm.rs

```diff
@@ -101,3 +101,62 @@ pub fn now() -> f64 {
         .expect("performance should be available")
         .now()
 }
+
+#[derive(Copy, Clone, Debug, PartialEq, PartialOrd)]
+pub struct SystemTime(f64);
+
+impl SystemTime {
+    pub const UNIX_EPOCH: SystemTime = SystemTime(0.0);
```

Comment:
I don’t think this is a valid definition. From what I understand, doing `SystemTime::now().duration_since(SystemTime::UNIX_EPOCH)` should return the duration since the unix epoch. However, the `now` function may [return results](https://github.com/sebcrozet/instant/blob/master/src/wasm.rs#L131-L142) from the JS function `performance.now()`, which is [not relative to the UNIX epoch](https://stackoverflow.com/a/30795586).

Reply:
What about https://github.com/wngr/instant/commit/bd46ccb577330d2d26ee40788e6439be6b488929 ? If @expenses agrees, I could create a new PR based on her commits.